tmt/utils/rest.py

```diff
@@ -49,17 +80,17 @@ class RestVisitor(docutils.nodes.NodeVisitor):
     node, filling "rendered paragraphs" list with rendered strings.
     """
 
+    def log_visit(self, node: Union[docutils.nodes.Node, docutils.nodes.Body]) -> None:
+        self.logger.debug('visit ', f'{node=}', level=4, topic=tmt.log.Topic.HELP_RENDERING)
+
+    def log_departure(self, node: Union[docutils.nodes.Node, docutils.nodes.Body]) -> None:
+        self.logger.debug('depart', f'{node=}', level=4, topic=tmt.log.Topic.HELP_RENDERING)
```

Comment:
```suggestion
        self.logger.debug('depart', f'{node}', level=4, topic=tmt.log.Topic.HELP_RENDERING)
```
Looks like a typo here.

Reply:
Fixed in badcaa8252814fb71a430663112bdfa0d0846d28